.github/workflows/botonic-cli-tests.yml

```diff
@@ -1,6 +1,12 @@
 name: Botonic cli tests
 
-on: [push]
+on:
+  push:
+    paths:
+      - '*'
```

Comment:
I guess that a single `*` is not recursive so it doesn't means a change anywhere